tests/test_indexer.py

```diff
@@ -146,6 +148,7 @@ def test_indexed_file_with_invalid_content_type(self):
                                                          files=smartseq2_paried_ends_indexed_file_list)
 
     def test_key_is_not_indexed_when_processing_an_event_with_a_nonbundle_key(self):
+        elasticsearch_delete_index(f"*{IndexSuffix.name}")
```

Comment:
Why do we delete the index at the start of a test?

Reply:
This is because the test was original designed expecting the indexes to be deleted at the beginning. I'll adjust the expected error message returned to better reflect what is being tested and removed the create and delete of indexes.